src/components/OrderPanel/BookingDatesForm/BookingDatesForm.js

```diff
@@ -499,6 +591,13 @@ export const BookingDatesFormComponent = props => {
         const endDatePlaceholderText =
           endDatePlaceholder || intl.formatDate(tomorrow, dateFormatOptions);
 
+        const realTimeSlots =
+          availabilityType === 'multipleSeats' && startDate
+            ? !endDate
+              ? combineConsecutiveTimeSlots(allTimeSlots, startDate)
+              : allTimeSlots
+            : allTimeSlots;
```

Comment:
Nested is difficult to understand, flatten this to (potentially chained) ternary:

```js
        const realTimeSlots =
          availabilityType === AVAILABILITY_MULTIPLE_SEATS && startDate && !endDate
            ? combineConsecutiveTimeSlots(allTimeSlots, startDate)
            : allTimeSlots;
```

Side note: I'm not sure but my gut feeling is that this should be extended to something like this:

```js
        const realTimeSlots =
          availabilityType === AVAILABILITY_MULTIPLE_SEATS && startDate && !endDate
            ? combineConsecutiveTimeSlots(allTimeSlots, startDate)
            : availabilityType === AVAILABILITY_MULTIPLE_SEATS && !startDate && endDate
            ? combineConsecutiveTimeSlots(allTimeSlots, endDate)
            : allTimeSlots;
```

Reply:
Renaming suggestion: realTimeSlots  => relevantTimeSlots